cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/StageContext.java

```diff
@@ -142,7 +143,9 @@
    *
    * @return the list of input stage names
    */
-  List<String> getInputStages();
+  default List<String> getInputStages() {
+    return new ArrayList<>();
```

Comment:
Is this breaking something?

Reply:
better to use java's collection api - `Collections.emptyList()`

Reply:
changed to use `Collections.emptyList()`. This is breaking https://builds.cask.co/browse/CDAP-BUT-2276/log. It broke a different repo (cdap-solutions/mmds), so I didn't notice it earlier.
https://github.com/cdap-solutions/mmds/blob/release/1.3/mmds-app/src/main/java/io/cdap/mmds/manager/WranglerContext.java#L45